pkgs/development/libraries/harfbuzz/default.nix

```diff
@@ -11,26 +11,27 @@
 , ninja
 , gobject-introspection
 , buildPackages
-, withIntrospection ? lib.meta.availableOn stdenv.hostPlatform gobject-introspection && stdenv.hostPlatform.emulatorAvailable buildPackages
+, withIntrospection ? if stdenv.hostPlatform.isMinGW then false else lib.meta.availableOn stdenv.hostPlatform gobject-introspection && stdenv.hostPlatform.emulatorAvailable buildPackages
```

Comment:
Can we instead modify meta.platforms of gobject-introspection to not include mingw?

Reply:
I'm dumb. gobject-introspection is already not supported on windows. This extra logic is unneeded.

Reply:
fixed

Reply:
fixed